lib/sigstore/models.rb

```diff
@@ -92,24 +93,55 @@ def find_rekor_entry(rekor_client)
 
       # debug
 
-      expected_entry = {
-        "spec" => {
-          "signature" => {
-            "content" => [signature].pack("m0"),
-            "publicKey" => {
-              "content" => [certificate.to_pem].pack("m0")
+      if signature
```

Comment:
this will be a lot clearer once I refactor the verifier to work directly on the sigstore protocol objects (a `dev.sigstore.verification.v1.Input`)